interfaces/builtin/basedeclaration.go

```diff
@@ -345,6 +345,10 @@ slots:
       slot-snap-type:
         - core
     deny-auto-connection: true
+  maliit-input-method:
+    allow-installation:
+      slot-snap-type:
+        - app
```

Comment:
I've set this to deny-connection: true and deny-auto-connection: true as only the Canonical patched version of maliit completely prevents sniffing and injection by checking with unity8 to determine whether the currently communicating app has focus.